assets/src/design-system/components/snackbar/snackbarMessage.js

```diff
@@ -178,8 +202,17 @@ SnackbarMessage.propTypes = {
   actionLabel: PropTypes.string,
   handleAction: PropTypes.func,
   isPreventAutoDismiss: PropTypes.bool,
+  isPreventActionDismiss: PropTypes.bool,
   removeMessageTimeInterval: PropTypes.number,
   showCloseButton: PropTypes.bool,
+  place: PropTypes.oneOf([
+    'top-left',
+    'bottom-left',
+    'top-right',
+    'bottom-right',
+    'top',
+    'bottom',
+  ]),
```

Comment:
nit: perhaps a constant in `snackbar/constants` to use where you reference strings above too?